packages/renderer/src/lib/ContainerDetailsLogs.svelte

```diff
@@ -90,13 +94,34 @@ async function refreshTerminal() {
   // call fit addon each time we resize the window
   window.addEventListener('resize', () => {
     if (currentRouterPath === `/containers/${container.id}/logs`) {
-      fitAddon.fit();
+      termFit.fit();
     }
   });
-  fitAddon.fit();
+  termFit.fit();
+}
+
+function resizeTerminal() {
+  if (logsTerminal) {
+    termFit.fit();
+  }
 }
+
 onMount(async () => {
+  // Refresh the terminal on initial load
   refreshTerminal();
+
+  // Resize the terminal each time we change the div size
+  resizeObserver = new ResizeObserver(entries => {
+    resizeTerminal();
```

Comment:
it seems the function is only called there, so probably we don't need the function and just call termFit?.fit();